lib/carin_for_blue_button_test_kit/custom_groups/visual_inspection_and_attestation/v200_client/must_support_process.rb

```diff
@@ -0,0 +1,26 @@
+module CarinForBlueButtonTestKit
+  class CARIN4BBMustSupportProcessTest < Inferno::Test
+    title 'Processes Must Support data elements'
+    description <<~DESCRIPTION
+      The Health IT Module must process all Must Support data elements without error.
+    DESCRIPTION
+    id :c4bb_must_support_process
+
+    verifies_requirements 'hl7.fhir.us.carin-bb_2.0.0@3'
```

Comment:
This is included in this test kit in both an automated test and here in an attestation test. Is that intentional?

Reply:
nope! removed attestation